kubeadm/v1.16.0/Kubeclustervxlan.json

```diff
@@ -0,0 +1,44 @@
+{
+    "Cri" : {
+        "Name" : "dockerd",
+        "Images" : {
+            "Pause" : "mcr.microsoft.com/k8s/core/pause:1.2.0",
+            "Nanoserver" : "mcr.microsoft.com/windows/nanoserver:1809",
```

Comment:
Do these need to be adjusted for Windows 1903?

Reply:
Yeah, but I think the default probably should stay at 1809?  The main changes just allow the script to work with any released version of windows as a worker node, but I think the default probably makes sense to stay at 1809.  Probably there should be documentation changes to tell people to use the right images for the right windows node OS version?

Alternately we could provide config files tailored for each OS release/kubernetes release, but that seems like it'd quickly get out of hand...

Reply:
Or another idea, we could remove the values from the config file for the default case and automatically pick the right images based on the host's OS version.  Still would be useful to have this as a configuration option for the user to be able to override the default versions (i.e. running against custom images or unsupported os versions), though.

Reply:
I see. I think the default values pointing to images that work on the LTS release is fine for now. Advanced/SAC users can override to the SAC versions as needed.